### ISSUE TITLE
Make use of a logging implementation or facade

### ISSUE THREAD

**Issue by CodeDead** [mkpaz/atlantafx#14]
Currently, errors are printed to the console, but perhaps it would be better to utilize a logging facade like SLF4J ?

**Comment by mkpaz** [comment]
Yes, it's in the backlog. Since there are no third-party libs, JUL should probably be enough. In normal mode, there's already [exception dialog](https://github.com/mkpaz/atlantafx/blob/master/sampler/src/main/java/atlantafx/sampler/DefaultExceptionHandler.java). I doubt that someone who just wants to see a quick feature preview need more than this. But log can be used in dev mode to print CSS errors when debugging a custom CSS theme.
